Docs/04-GeneratingCertificates.md

```diff
@@ -0,0 +1,103 @@
+# Generating Certificates
+
+This document will walk you through one way to generate a CA and self-signed certificate for use with Steward.
+
+Two-way trust must be established between Steward and the Strategy Provider client for each request. To accomplish this, both the Server and Client will need TLS certificates and the CA certificate of the other party (Steward needs the client's CA, and the client needs Steward's CA). To make generating Steward's certificates easier, we've provided a convenience script called `gen-script.sh` which can be found [here](https://github.com/peggyjv/steward/gen-script.sh). The script takes one optional flag for the output location of the certificates and their signing keys. The client CA will be set by default for now/
```

Comment:
Nit: trailing `/`. Also: default to what? Perhaps saying PeggyJV's CA at initial launch.